dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINAlgRestoration.cpp

```diff
@@ -237,6 +235,32 @@ SolverKINAlgRestoration::evalF_KIN(N_Vector yy, N_Vector rr, void *data) {
   return (0);
 }
 
+void
+SolverKINAlgRestoration::checkJacobian(const SparseMatrix& smj, const boost::shared_ptr<Model>& model) {
```

Comment:
Please add this method only in DEBUG

Reply:
will it not be automatiicaly remove at compilation ?

Reply:
I'd prefer to avoid use the flag #if _DEBUG_ in a lot of places

Reply:
we have to compile on many versions of gcc/clang and OS so we cannot make any assumption about the compiler optimizations, and also this is to point out to developers that this is a debug method

Reply:
OK